eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -25,6 +25,20 @@
     public $identifier;
 
     /**
+     * True before the role has been published.
+     *
+     * @var bool
+     */
+    public $is_new = true;
```

Comment:
I think that historically drafts might have been implemented for contenttypes because of the fact that the editing GUI was implemented via a multi-submit form (add a field, configure it, add another one, etc... then at the end click 'apply' and the existing contents get updated), whereas actions on other entities where applied atomically, with the whole entity being saved each time it is modified (there is no save-draft button when editing roles and policies). Iirc workflows also have a draft state, and they also have a multi-step editing process.

Since the idea for eZPlatform is not to be bound by
- legacy data structures
- any single GUI process
I think that the draft thing could be rethought (now or later) with a bit of a more consistent approach.